.travis-addon.yml.example

```diff
@@ -3,12 +3,8 @@ python:
   - "3.6"
 
 install:
-  - pip install Pillow requests
-
-before_install:
-  - git submodule update --init --recursive --remote
+  - pip install Pillow requests kodi-addon-checker
```

Comment:
Though it is perfectly fine to install Pillow and requests, it is not necessary to do so.

You can simply call `pip install kodi-addon-checker` which will install `kodi-addon-checker` along with `Pillow` and `requests` because they are already marked as dependencies in [setup.py#L5](https://github.com/xbmc/addon-check/blob/master/setup.py#L5)

Reply:
makes sense, forgot about that when adding the new dependency